debugger/lldbPythonExtension/lldb_sgx_plugin.py

```diff
@@ -0,0 +1,321 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from __future__ import print_function
+import struct
+import os.path
+import os
+import lldb
+from ctypes import create_string_buffer
+import load_symbol_cmd
+import ctypes, mmap, errno
+from threading import Thread, Lock
+
+
+POINTER_SIZE = 8
+
+# These constant definitions must align with _oe_enclave structure defined in host\enclave.h
+OE_ENCLAVE_MAGIC_FIELD = 0
+OE_ENCLAVE_ADDR_FIELD = 2
+OE_ENCLAVE_HEADER_LENGTH = 0X28
+OE_ENCLAVE_HEADER_FORMAT = 'QQQQQ'
+OE_ENCLAVE_MAGIC_VALUE = 0x20dc98463a5ad8b8
+
+OE_ENCLAVE_FLAGS_OFFSET = 0x588
```

Comment:
OE_ENCLAVE_FLAGS_OFFSET is incorrect and should be 0x598. This has since been fixed in the master and due to changes in the _oe_enclave_structure.
To avoid duplication between gdb and lldb, these constant definitions and common routines should be put into a common library and used by both.